rocket_sqlite_oracle/dev_keys/oracle_secret

```diff
@@ -0,0 +1 @@
+e�3��	�����U��׽V#���͒c�e6(
```

Comment:
Shouldn't check this into VCS :)

Reply:
Clearly :-) But this was done intentionally for the sake of testing (not done) / multiple developers being able to obtain consistent results.

Reply:
We don't need to test for the exact signature, that is not our code and hence not our testing responsibility. As a result, which key is used for testing doesn't matter :)